app/controllers/distributions_controller.rb

```diff
@@ -105,7 +105,11 @@ def update
 
     if result.success?
       if result.resend_notification? && @distribution.partner&.send_reminders
-        send_notification(current_organization.id, @distribution.id, subject: "Your Distribution New Schedule Date is #{@distribution.issued_at}")
+        if result.issued_at_changed?
+          send_notification(current_organization.id, @distribution.id, subject: "Your Distribution's new Schedule Date is #{@distribution.issued_at}")
+        elsif result.delivery_method_changed?
+          send_notification(current_organization.id, @distribution.id, subject: "Your Distribution's new Delivery Method is #{@distribution.delivery_method.humanize}")
+        end
```

Comment:
what if both change?

Reply:
oh nice catch! Yeah we could do something more generic.